processing/src/main/java/io/druid/segment/BitmapOffset.java

```diff
@@ -40,6 +40,21 @@
 
   private volatile int val;
 
+  public static IntIterator getReverseBitmapOffsetIterator(ImmutableBitmap bitmapIndex)
+  {
+    ImmutableBitmap roaringBitmap = bitmapIndex;
+    if (!(bitmapIndex instanceof WrappedImmutableRoaringBitmap)) {
+      final BitmapFactory factory = RoaringBitmapSerdeFactory.bitmapFactory;
```

Comment:
this seems pretty expensive if the bitmap isn't roaring

are we planning on making roaring the default?